rewrite-maven/src/main/java/org/openrewrite/maven/UpgradeDependencyVersion.java

```diff
@@ -402,6 +407,28 @@ private String resolveVersion(String version) {
                 return MavenDependency.findNewerVersion(groupId, artifactId, version, getResolutionResult(), metadataFailures,
                         versionComparator, ctx);
             }
+
+            /**
+             * Gets the requested group id for the provided artifact id from the pom
+             *
+             * @return The groupd if for the requested artifact id or null
+             */
```

Comment:
The comments here seem to just echo the method name & types. Lets drop rather than maintain those.
```suggestion
```